packages/orchestrator-ui-components/src/components/WfoAuth/WfoAuth.tsx

```diff
@@ -1,22 +1,31 @@
 import React, { JSX, useContext } from 'react';
 
-import { useSession } from 'next-auth/react';
-
 import { WfoLoading } from '@/components';
 import { OrchestratorConfigContext } from '@/contexts';
+import { PolicyContextProvider } from '@/contexts/PolicyContext';
+import { useWfoSession } from '@/hooks';
 
 interface AuthProps {
     children: JSX.Element;
+    isAllowed?: (routerPath: string, resource?: string) => boolean;
 }
 
-export const WfoAuth = ({ children }: AuthProps): JSX.Element => {
+export const WfoAuth = ({
+    children,
+    isAllowed = () => true,
```

Comment:
add documentation for implementing isAllowed.

Reply:
Done, added it above the WfoAuth component declaration